src/api.js

```diff
@@ -4,15 +4,14 @@ const api = axios.create({
   baseURL: "https://be-nc-news-ml9n.onrender.com/api",
 });
 
-function getArticles(topic) {
+function getArticles(queryString) {
   let endpoint = "/articles"
-  if(topic){
-    endpoint += `?topic=${topic}`
-  }
+  endpoint += queryString
```

Comment:
Look at the axios docs for the params object, it might help you here

Reply:
I'll look into that cheers!